doc/components_1.md

```diff
@@ -27,15 +28,97 @@ For instance, the `mj-button` component is, on the inside, a complex HTML layout
 </table>
 ```
 
+#### Attribute lists
+
+MJML authors use _attributes_ to specify custom behavior of MJML components.
+Our documentation has _attribute lists_ for each component describing each 
+  supported attribute.
+The lists have four columns.
+The first attribute list is just below, under the ["mjml" heading](##mjml).
+
+The _attribute_ column is a complete list of attributes a component supports.
+An examples of syntax for using a attributes appears just below here
+  in discussion of the `mjml` tag and its `lang` attribute.
+
+<aside class="notice">
+In general, if MJML has an attribute of the same name as an HTML attribute,
+  the two operate similarly.
+</aside>
+
+<!--- NOTE: I understand Markdown recognizes this three-dash comment
+  syntax as a line that should appear in the HTML as a comment (two-dash),
+  but should not render in the documentation. Garry votes that's okay.
+  There's advantage in putting a comment here for future documentation
+  maintainers. --> 
+<!--- Example 1: MJML's type "color" (color.js in mjml-core/src/types) supports notations
+  (1) #RRGGBB, (2) #RGB, (3) rgb(), and (4) rgba(), and no other.
+  HTML also supports "hsl()" notation and others. Still, they operate similarly. -->
+<!--- Example 2:
+  MJML's "mj-text" > "padding" limits the attribute values to "unit(px,%,)".
+  HTML's "padding" also supports "em" units and
+  three <em>global value</em> strings. Still similar. -->
+  
+The _unit_ column specifies whether the attribute must be, for instance,
+  a string, a number, or other unit.
+There's a little more on this later in this section.
+
+The _description_ column explains the purpose of the attribute
+  and what the developer can achieve with it.
+Sometimes, it indicates equivalence with the same-name HTML attribute. 
+
+The _default value_ column specifies the default value for the attribute.
+Attributes without a default value have _n/a_ here.
+
+<!--- Not ambiguous because MJML does not support "n/a" as any attribute value. --> 
+
+<aside class="notice">
+MJML syntax for <em>HTML</em> attributes differs from syntax for <em>MJML</em> attributes.
+MJML "css-class" attributes introduce HTML attributes in MJML components;
+  they refer to [mj-style](#mj-style) definitions.
+MJML attributes use the same syntax in MJML tags as HTML attributes in HTML tags.
+</aside>
+
+<aside class="notice">
+MJML can support both an HTML attribute and an MJML attribute
+  with the same spelling and different values.
+In such code, MJML uses the HTML attribute
+  inside an MJML "css-class" attribute and
+  in raw HTML inside a "class" attribute.
+It uses the MJML attribute otherwise.
+</aside>
+
+<!--- https://mjml.io/try-it-live/H1LbbsI1I -->
+
+<aside class="notice">
```

Comment:
Yeah not sure about this paragraph, it's a bit weird to say this where we try to solve that issue with MJML.

We could add a list of resource to check css compatibility like https://www.caniemail.com/

Reply:
Fair. I'll make the case for leaving this in, but I'm happy to remove it. I certainly agree MJML protects the developers from these problems in many cases (hopefully most or almost all).

That said, we know Outlook doesn't support padding well. Or media queries. GMail doesn't support fonts. And ... you know far better than I.

I'll add the caniemail.com reference. And plug MJML as helping handle some of the problems. Maybe that'll add appropriately.

Reply:
I think a simple warning that you should still check if your CSS attributes/values are supported everywhere before using them and you can check with caniemail.

Reply:
Lines 75-78.